### ISSUE TITLE
[Feature request] Is it possible to move dist to libs?

### ISSUE THREAD

**Issue by cx20** [Reon90/redcube#16]
Currently, [gltf-test](https://github.com/cx20/gltf-test/) has the following folder structure.
https://github.com/cx20/gltf-test/

````
/examples
    /redcube
        /index.html
        /dist
            /xxxxx.hdr
/libs
    /redcube
         /x.x.x
             /redcube.js
````

However, currently, the library that `Redcube.js` depends on externally does not seem to work unless it is placed in the `dist` folder under the `examples` folder.
Is it possible to move this under the `libs` folder?

Also, since it is difficult to understand what kind of function the file name composed of numerical values is, is it possible to have the name easy to understand?

**Comment by Reon90** [comment]
@cx20 Implemented in 2.5.2.

**Comment by cx20** [comment]
@Reon90 Thank you for changing the file name to something more understandable.

However, it seems that the files under the `dist` folder cannot be referenced unless they are placed in the same folder as `index.html`. It may be a matter of taste, but it would be nice to be able to put them in a separate folder.

My hope is that the [gltf-test](https://github.com/cx20/gltf-test) folder structure looks like this:
```
/examples
    /redcube
        /index.html
        /index.js
/assets 
    /Sheen_E.hdr
/libs
    /redcube
        /x.x.x
            /redcube.js
            /libs
                /draco3d.js
                /draco_decoder.wasm
                /libktx.js
                /libktx.wasm
```

**Comment by Reon90** [comment]
@cx20 fixed in 2.5.9

**Comment by cx20** [comment]
I am grateful a lot.
